src/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from './ppom-controller';
 
-export type { StorageBackend, StorageKey } from './ppom-storage';
+export type { StorageKey } from './ppom-storage';
```

Comment:
I think we do not need to export even `StorageKey` any longer now as it will not be required by extension or mobile code now.